doc/classes/Time.xml

```diff
@@ -85,8 +85,8 @@
 			<param index="1" name="use_space" type="bool" default="false" />
 			<description>
 				Returns the current date and time as an ISO 8601 date and time string (YYYY-MM-DDTHH:MM:SS).
-				The returned values are in the system's local time when [code]utc[/code] is false, otherwise they are in UTC.
-				If [code]use_space[/code] is true, use a space instead of the letter T in the middle.
+				The returned values are in the system's local time when [param utc] is false, otherwise they are in UTC.
+				If [param use_space] is true, use a space instead of the letter T in the middle.
```

Comment:
```suggestion
				If [param use_space] is [code]true[/code], the date and time bits are separated by an empty space character instead of the letter T.
```